test/functional/lua/vim_spec.lua

```diff
@@ -1817,7 +1817,7 @@ describe('lua stdlib', function()
       local function get_counter() return counter end
       vim.t.AddCounter = add_counter
       vim.t.GetCounter = get_counter
-      vim.api.fn = {add = add_counter, get = get_counter}
+      vim.t.fn = {add = add_counter, get = get_counter}
```

Comment:
?

Reply:
What's the problem?

Reply:
I thought this was maybe an accidental search-replace. But now I see this might be *fixing* an accidental replace.